flink-examples/doc/flink-high-error-count-alert/README.md

```diff
@@ -0,0 +1,133 @@
+# High Count Alert #
+
+The application reads apache access logs from a Pravega stream and once every 2 seconds
+counts the number of 500 responses in the last 30 seconds, and generates
+alert when the counts of 500 responses exceed 6.
+
+## Prerequistes ##
+
+A Docker image containing Pravega and Logstash had been prepared to simplify the demo. Skip ahead to the **Run in Docker Container** section in this document if you have docker environment handy.
+
+Otherwise proceed to set up Logstash and Pravega
+
+1. Logstash installed, see [Install logstash](https://www.elastic.co/guide/en/logstash/5.6/installing-logstash.html).
+2. Pravega running, see [here](http://pravega.io/docs/latest/getting-started/) for instructions.
+
+## Start Logstash with Pravega Output Plugin ##
+
+On the Logstash host, download the plugin gem file from [Logstash Pravega output plugin](https://github.com/pravega/logstash-output-pravega/releases), for example, `logstash-output-pravega-0.2.0.gem`.
+
+Install the plugin, assuming Logstash is installed at `/usr/share/logstash/`
+```
+$ /usr/share/logstash/bin/logstash-plugin install logstash-output-pravega-0.2.0.gem
+```
+
+Copy the contents under flink-examples/doc/flink-high-error-count-alert/filters/ to the Logstash host, e.g., in directory ~/pravega.
+update **pravega_endpoint** in ~/pravega/90-pravega-output.conf
+
+```
+output {
+    pravega {
+        pravega_endpoint => "tcp://127.0.0.1:9090"   <- update to point to your Pravega controller
+        stream_name => "apacheaccess"
+        scope => "examples"
+    }
+}
+```
+
+Start logstash, assuming it is installed at /usr/share/logstash/bin.
+Note that sometimes it may take a minute or two for logstash to start. For troubleshooting, the logstash log files are 
+normally at /var/log/logstash. To restart, type Ctrl-C, then re-run the command.
+
+```
+$ sudo /usr/share/logstash/bin -f ~/pravega
+Sending Logstash's logs to /var/log/logstash which is now configured via log4j2.properties
+```
+
+Normally Logstash is configured to receive data from remote log shippers, such as filebeat. For simplicity in this demo
+Logstash is configured read data from /tmp/access.log.
+
+## Run in Docker Container ##
+
+Create a file at /tmp/access.log
+```
+$ touch /tmp/access.log
+```
+
+Run script below to start container from prebuilt image. Adjust parameters to your need.
+```
+#!/bin/sh
+set -u
+
+PRAVEGA_SCOPE=myscope
+PRAVEGA_STREAM=apacheaccess
+CONTAINER_NAME=pravega
+IMAGE_NAME=emccorp/pravega-demo
+
+docker run -d --name $CONTAINER_NAME \
+    -p 9090:9090 \
+    -p 9091:9091 \
+    -v /tmp/access.log:/opt/data/access.log \
+    -v /tmp/logs/:/var/log/pravega/ \
+    -e PRAVEGA_ENDPOINT=${PRAVEGA_ENDPOINT} \
```

Comment:
Please document what values should be passed (default?) to `PRAVEGA_ENDPOINT` variable?